src/screens/Nearby/NearbyResults.tsx

```diff
@@ -52,10 +55,16 @@ const NearbyResults: React.FC<NearbyResultsProps> = ({
 }) => {
   const styles = useResultsStyle();
   const navigation = useNavigation<NearbyScreenNavigationProp>();
+  const {t} = useTranslation();
   const onPress = (departure: EstimatedCall) => {
     const {publicCode, name} = getLineNameFromEstimatedCall(departure);
+    const title = publicCode
+      ? `${publicCode} ${name}`
+      : name
+      ? name
+      : t(dictionary.travel.line.defaultName);
     navigation.navigate('DepartureDetailsModal', {
-      title: publicCode ? `${publicCode} ${name}` : name,
+      title: title,
```

Comment:
suuuuuuuuuper nitpicky stuff
```suggestion
      title,
```

Reply:
Yes, but really non-pleasing to the eye :grimacing: Some automatic refactoring gone wrong